README.md

```diff
@@ -465,6 +465,8 @@ The following `opts` are supported:
 - `opts.markers`     : show markers (`boolean`; default = `false`)
 - `opts.markersymbol`: marker symbol (`string`; default = `'dot'`)
 - `opts.markersize`  : marker size (`number`; default = `'10'`)
+- `opts.linecolor`   : line colors (`np.array`; default = None)
+- `opts.dash`        : line dash type for each line (`np.array`; default = None), one of `solid`, `dash`, `dashdot` or `dash`, size should match number of lines being drawn
```

Comment:
Nit: We should note that defaulting to none will give all lines 'solid' style.

Reply:
The plotly implementation actually defaults to 'solid' if given None, but it's a good idea to make it more explicit here. I'll change it 👍